AutoMapper.Data/ConfigurationExtensions.cs

```diff
@@ -10,8 +11,8 @@ public static void AddDataReaderMapping(this IMapperConfigurationExpression conf
 
         public static void AddDataReaderMapping(this IMapperConfigurationExpression configuration, bool enableYieldReturn)
         {
-            configuration.Mappers.Insert(0, new DataReaderMapper { YieldReturnEnabled = enableYieldReturn });
-            configuration.AddMemberConfiguration().AddMember<DataRecordMemberConfiguration>();
+            InternalApi.Internal(configuration).Mappers.Insert(0, new DataReaderMapper { YieldReturnEnabled = enableYieldReturn });
```

Comment:
`AddDataReaderMapping()` or the extension method `Internal()`, _everywhere_ :)

Reply:
Ugh, yeah, thanks for the notice

Reply:
Checked all the places, but didn't find any gaps. May be I didn't get your point correctly 😕
Everywhere either `AddDataReaderMapping()` or the extension method `Internal()` is used.

Reply:
Fixed!